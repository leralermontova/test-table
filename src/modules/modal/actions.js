import { createAction } from 'redux-act';

export const open = createAction(
  'OPEN_MODAL',
);

export const close = createAction(
  'CLOSE_MODAL',
);
