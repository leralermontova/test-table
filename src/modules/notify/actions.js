import { createAction } from 'redux-act';

export const show = createAction(
  'SHOW_NOTIFY',
);
