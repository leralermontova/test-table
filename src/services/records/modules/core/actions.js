import { createAction } from 'redux-act';

export const createSuccess = createAction(
  'RECORDS_CORE_CREATE_SUCCESS',
);

export const create = createAction(
  'RECORDS_CORE_CREATE',
  record => ({
    promise: client => client.post('/record', record),
  }),
);

export const load = createAction(
  'RECORDS_CORE_GET',
  () => ({
    promise: client => client.get('/records'),
  }),
);
