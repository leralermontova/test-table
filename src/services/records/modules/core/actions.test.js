import { createSuccess } from './actions';

describe('modal actions', () => {

  it('should create correctly', () => {

    expect(createSuccess()).toMatchSnapshot();

  });

});
