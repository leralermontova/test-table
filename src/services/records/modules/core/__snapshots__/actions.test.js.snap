// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal actions should create correctly 1`] = `
Object {
  "error": false,
  "payload": undefined,
  "type": "RECORDS_CORE_CREATE_SUCCESS",
}
`;
